app/models/user.rb

```diff
@@ -0,0 +1,14 @@
+class User < ApplicationRecord
+  has_many :posts, foreign_key: :author_id
+  has_many :comments, foreign_key: :author_id
+  has_many :likes, foreign_key: :author_id
+
```

Comment:
- [ ] It is only the Posts table that has `author_id` column, the `Comments` and `Likes` table don't have the `author_id` column. They are tied to the `Users` table through the `user_id` column. Kindly implement this change. 👍 😎

Reply:
Fixed.